jsconfig.json

```diff
@@ -3,7 +3,8 @@
     "baseUrl": ".",
     "paths": {
       "~/*": ["./*"]
-    }
+    },
+    "jsx": "preserve"
```

Comment:
What does this do?

Reply:
Actually Volar was throwing the following warning. So either I need to turn off Volar and enable Vetur since our websites still use Vue 2 or add this option jsx: preserve  in tsconfig or jsconfig. I added this option in jsconfig since we all are using Volar now

![Screenshot from 2022-05-30 10-10-14](https://user-images.githubusercontent.com/24249988/170922120-3db93e4b-a5c2-4d6c-8f67-8a21949ac0aa.png)